posthog/migrations/0256_pluginstorage_timestamp.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.14 on 2022-08-16 15:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    atomic = False
+
+    dependencies = [
+        ("posthog", "0255_user_prompt_sequence_state"),
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(model_name="pluginstorage", name="posthog_unique_plugin_storage_key",),
+        migrations.AddField(
+            model_name="pluginstorage", name="timestamp", field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.RunSQL(
+            "CREATE INDEX CONCURRENTLY IF NOT EXISTS pluginstorage_timestamp_date_idx ON posthog_pluginstorage(timestamp);",
```

Comment:
I _believe_ this will time out on prod, creating a potential outage - create index still is a blocking operation and we have a pretty strict time limit set on queries.